main.go

```diff
@@ -46,7 +46,7 @@ func main() {
 
 	initHttpHandlers()
 
-	sessionObj := make(chan utils.SessionObj)
+	sessionObj := make(chan utils.SessionObj, 50)
```

Comment:
why 50? and why not in a const

Reply:
I spoke with @rcohencyberarmor  and this is the number we came up with in order not to take too much memory but also not to block too much when inserting